.github/workflows/build.yaml

```diff
@@ -282,7 +282,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        tag_suffix: ["", "-fast", "-fast-slim"]
```

Comment:
@tommartensen FYI, I removed `check-collector-images-exist (-fast-slim)` from required checks for the `master` branch.